test/spec/unit/pbjs_api_spec.js

```diff
@@ -1046,7 +1056,7 @@ describe('Unit: Prebid Module', function () {
       })
 
       it('should callBids if a native adUnit has all native bidders', () => {
-        // TODO: appnexusAst is currently hardcoded in native.js, update this text when fixed
+        // TODO: appnexus is currently hardcoded in native.js, update this text when fixed
```

Comment:
is this comment true? if it's possible fix it or drop if it's not true.

Reply:
Todo might have been completed long back so dropped the comment